### DISCUSSION TITLE
Commits not being shown in GitHub profile diagram

### DISCUSSION THREAD

**Discussion by RobertAliaj** [community/community#47894] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi everyone, I have been making several commits on my GitHub profile, but only a few of them are being shown in the contribution diagram. I always use the following Git commands to commit and push my changes: 

1. git add . 
1. git commit -m 'commit'
3. git push

I'm not sure what else could be causing this issue. Any suggestions or solutions would be greatly appreciated. Thank you!

**Comment by airtower-luna** [comment]
Without seeing the repository it's impossible to answer, but you can see the conditions under which a commit shows up in the graph in the documentation: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile

Most likely one of those conditions isn't met.

**Comment by OwaisAzad** [comment, marked as answer]
There are might be a few reasons why your commits not showing:

Commits to forked repositories: If you have been contributing to a repository that you have forked, your contributions may not show up on your contribution graph by default.

Commits made with a different email address: GitHub uses the email address associated with your Git commits to attribute contributions to your profile. If you have been making commits with a different email address than the one associated with your GitHub account, these contributions may not be attributed to your profile.

Commits made in a different time zone: GitHub calculates contributions based on Coordinated Universal Time (UTC). If you have been making commits in a different time zone, the contributions may not be attributed to the correct day on the contribution graph.

**Reply by RobertAliaj** [reply]
It was because i had an other Email, which i don't know why it was there. But anyway i got it now, Thank you:)

**Reply by mkos11** [reply]
Certainly! If you're experiencing issues with your commits not showing up on your GitHub contribution graph, here are a few steps you can take to address the potential reasons mentioned:

1. **Check Forked Repositories:**
   - If you've been contributing to a repository that you forked, your contributions may not be visible by default. Ensure that you are checking the contribution graph on your forked repository as well.

2. **Verify Email Address:**
   - GitHub attributes contributions based on the email address associated with your Git commits. Confirm that the email address in your Git configuration matches the one linked to your GitHub account.

3. **Time Zone Considerations:**
   - GitHub calculates contributions in Coordinated Universal Time (UTC). Double-check the time zone settings on your local machine to ensure your commits are attributed to the correct day on the contribution graph.

If you've addressed these aspects and are still encountering issues, you may want to provide more specific details about your situation, such as the repository you're contributing to and any error messages you're receiving. This will help in providing more targeted assistance.

**Comment by giginar** [comment]
Hello @OwaisAzad , I have the same issue on this repo. https://github.com/giginar/Reddit-Clone

although, I am pushing some stuff last two days its now shown.

Thank you.
